packages/common/src/evo/data_converters/common/objects/base_properties.py

```diff
@@ -0,0 +1,37 @@
+import typing
+from abc import ABC
+
+
+class BaseObjectProperties(ABC):
+    def __init__(
+        self,
+        *,
+        name: str,
+        uuid: str | None = None,
+        description: str | None = None,
+        extensions: dict[str, typing.Any] | None = None,
+        tags: dict[str, str] | None = None,
+    ) -> None:
+        self.name: str = name
+        self.uuid: str | None = uuid
+        self.description: str | None = description
+        self.extensions: dict[str, typing.Any] | None = extensions
+        self.tags: dict[str, str] | None = tags
+
+
+class BaseSpatialDataProperties(BaseObjectProperties):
+    def __init__(
+        self,
+        *,
+        name: str,
+        uuid: str | None = None,
+        coordinate_reference_system: int | str | None = None,
+        description: str | None = None,
+        extensions: dict[str, typing.Any] | None = None,
+        tags: dict[str, str] | None = None,
+    ) -> None:
+        super().__init__(name=name, uuid=uuid, description=description, extensions=extensions, tags=tags)
+        self.coordinate_reference_system: int | str | None = coordinate_reference_system
```

Comment:
Noticed after chat discussion - this maybe should be:
```suggestion
        self.coordinate_reference_system: int | str = coordinate_reference_system or "unspecified"
```

Reply:
that's pretty swish, I'll make that change thanks :+1: